packages/env/lib/wordpress.js

```diff
@@ -51,11 +91,33 @@ async function checkDatabaseConnection( { dockerComposeConfigPath, debug } ) {
  * @param {Object}        spinner     A CLI spinner which indicates progress.
  */
 async function configureWordPress( environment, config, spinner ) {
+	let wpVersion = '';
+	try {
+		wpVersion = readWordPressVersion(
+			config.env[ environment ].coreSource,
+			spinner,
+			config.debug
+		);
+
+		// Sometimes wpVersion may not be a string, in this case, convert it to a string.
+		if ( typeof wpVersion !== 'string' ) {
+			wpVersion = '' + wpVersion;
+		}
```

Comment:
From what I can see, this is impossible.

`readWordPressVersion()` returns a regex match from a file. It will always be a string.

Reply:
I thought so too, but check https://github.com/WordPress/gutenberg/actions/runs/10376725339/job/28729310832. I wonder if the regex match is somehow wrapped in something else that mostly works like a string, but isn't actually a string.

Reply:
Huh 🤔  this doesn't make any sense. Have you done any debugging to see what the value of `wpVersion` actually was?

If for some reason it may not be a string, then casting should be better done in `readWordPressVersion`.

`'' +` is not really a technique used elsewhere in the code base, and especially for novices it might not be obvious why it works (the + operator in JS has its ... perks). Might be better to do `wpVersion.toString()` or

```
`${ wpVersion }`
```